lib/vanagon/platform/windows.rb

```diff
@@ -53,9 +53,21 @@ def generate_packaging_artifacts(workdir, name, binding)
       # @param name [String] name of the project
       # @param binding [Binding] binding to use in evaluating the packaging templates
       def generate_msi_packaging_artifacts(workdir, name, binding)
-        FileUtils.mkdir_p(File.join(workdir, "wix"))
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.wxs.erb"), File.join(workdir, "wix",  "#{name}.wxs"), false, { :binding => binding })
-        erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.filter.xslt.erb"), File.join(workdir, "wix", "#{name}.filter.xslt"), false, { :binding => binding })
+        # Copy (cp -r) files to workdir/wix folder. The VANAGON_ROOT root files are copied first, followed by the project specific files.
+        # The "remove_destination" flag is set which ensures that project specific files will always over-write generic/vanagon files
+        FileUtils.cp_r(File.join(VANAGON_ROOT, "resources/windows/wix"), workdir, :verbose => true, :remove_destination => true)
+        FileUtils.cp_r("resources/windows/wix", workdir, :verbose => true, :remove_destination => true)
+        # Evaluate and process all *.erb files found in workdir/wix
+        # The originating .erb file is left in place as it is ignored by the wix/candle process.
+        Dir.chdir(File.join(workdir, "wix")) do
```

Comment:
Actually thinking about it - since all we are doing is stripping off the .erb for the destination - the manipulation is straightforward so will take out the chdir.